### ISSUE TITLE
Expo SDK 34 error with FileSystem.writeAsStringAsync

### ISSUE THREAD

**Issue by jasonkv** [expo/expo-three#116]
Hi,
We are trying to upgrade to Expo 34 (to get now required play store 64 bit app). We are seeing this error that seems to be caused by:

`import ExpoTHREE from "expo-three";
`
The error is:

`undefined is not an object (evaluating '_expo.FileSystem.writeAsStringAsync')
`

Stack trace start:
```
<unknown>
C\User\....app_path...\node_modules\@expo\browser-polyfill\src\DOM\HTMLImageElement.js3:8

```

package.json:

`    "expo-three": "^4.0.3",
`

Any ideas?

**Comment by jasonkv** [comment]
Pulled down this PR and that gets the error to go away but now getting a new problem with trying to load obj files:
https://github.com/expo/expo-three/pull/115
The new error is:

ExpoTHREE.loadAsync(require('someObjFile'));

` [TypeError: undefined is not a constructor (evaluating 'new _Three.default.OBJLoader()')]
`

**Comment by plaidpowered** [comment]
I'm also experiencing this issue, on a brand new project. Are there any workarounds?

**Comment by plaidpowered** [comment]
Discovery: If I create a new project and install using `npm install three expo-three`, this error occurs. If I install the packages using yarn instead, I have no issue. Hope this helps someone.

**Comment by jasonkv** [comment]
> Discovery: If I create a new project and install using `npm install three expo-three`, this error occurs. If I install the packages using yarn instead, I have no issue. Hope this helps someone.

Interesting. I wonder what the difference is in your package.json and lock files.

I deleted my node_modules folder and did an `npm install` and I believe the problem has been in a recent commit.

**Comment by plaidpowered** [comment]
I've temporarily switched my repositories to public so you can compare them (it's just demo/getting-started code right now anway)
The working repo, that I built with yarn, is here: https://github.com/plaidpowered/wp-native
You'll note it doesn't have a package-lock.json committed, there isn't one.

The non-working repo, built purely with npm, is here: 
https://github.com/plaidpowered/pi-native

**Comment by jasonkv** [comment]
> I've temporarily switched my repositories to public so you can compare them (it's just demo/getting-started code right now anway)
> The working repo, that I built with yarn, is here: https://github.com/plaidpowered/wp-native
> You'll note it doesn't have a package-lock.json committed, there isn't one.
> 
> The non-working repo, built purely with npm, is here:
> https://github.com/plaidpowered/pi-native

Might be that the versions are different? Makes sense since one seems to be older. But I am not sure which it is actually installing for each

**Comment by EvanBacon** [comment]
Fixed in latest 5.5.1
